### DISCUSSION TITLE
Add padding option in settings

### DISCUSSION THREAD

**Discussion by NicKoehler** [linuxdeepin/developer-center#2973] [Features Request & Ideas | 特性请求 & 头脑风暴]
Please consider add padding setting option for making the text more readable, like the example screenshot attached

![image](https://user-images.githubusercontent.com/53040044/103481593-47579c80-4ddc-11eb-9120-a3bca30e7950.png)

**Comment by wangliang-terminal** [comment]
Which terminal in the screenshot are you using? I'd like to try it out and see how he does it.

**Comment by NicKoehler** [comment]
> Which terminal in the screenshot are you using? I'd like to try it out and see how he does it.

This is "gnome-terminal", you can get that padding in GTK-based terminals editing `~/.config/gtk-3.0/gtk.css`, meanwhile for kde default terminal "konsole" you need to edit the profile in "`~/.local/share/konsole/`" and add the line:
`TerminalMargin=7`
Under the `[General]` section.

**Comment by ch593030323** [comment]
Thank you for your suggestion. We will consider this function reasonably.
